main/guides/governance/index.md

```diff
@@ -0,0 +1,263 @@
+
+export const start = async (zcf, privateArgs, baggage) => {
+...
+  const { publicMixin, makeDurableGovernorFacet, params } =
+    await handleParamGovernance(
+      zcf,
+      privateArgs.initialPoserInvitation,
+      paramTypes,
+      privateArgs.storageNode,
+      privateArgs.marshaller,
+    );
+...
+}
+```
+
+We get back
+
+- `params`: read-only parameter access. `params.getFee().value` gives us the current value of the `Fee` parameter at any time, for example.
+- `publicMixin`: to make the parameter values available via the contract `publicFacet`
+- `makeDurableGovernorFacet`: to combine the contract's existing creator facet methods (known as the `limitedCreatorFacet`) with methods for _changing parameter values_.
+
+```js
+export const start = async (zcf, privateArgs, baggage) => {
+...
+  const publicFacet = Far('Public', {
+    makeFirstInvitation,
+    ...publicMixin,
+  });
+  const limitedCreatorFacet = Far('Creator', {
+    makeCollectFeesInvitation() {
+      return makeCollectFeesInvitation(zcf, feeSeat, feeBrand, 'Fee');
+    },
+  });
+  const { governorFacet } = makeDurableGovernorFacet(
+    baggage,
+    limitedCreatorFacet,
+  );
+  return harden({ publicFacet, creatorFacet: governorFacet });
+}
+```
+
+## Starting a Governed Contract via its Governor
+
+Changing parameter values is something only the electorate is authorized to do.
+To enforce this, a governed contract is started by a _contract governor_.
+The contract governor holds the fully-functional creator facet that includes
+the capability to change the parameters. The caller that starts the governor
+gets only the `limitedCreatorFacet`.
+
+<img src="./assets/gov-param-2.svg"
+  style="border: 2px solid" width="600" />
+
+_For clarity, some Zoe API details are ommitted from this figure._
+
+## Putting a question via an Election Manager
+
+An _ElectionManager_ is responsible for letting an appropriate party call `addQuestion()`. The `econCommitteeCharter.js` contract from `@agoric/inter-protocol` is sufficiently general to handle most forms of parameter
+governance, so we reuse it here. Swaparoo core eval deployment likewise
+starts this contract, asks it for invitations, and sends them to the voters:
+
+<img src="./assets/gov-param-3.svg"
+  style="border: 2px solid" width="600" />
+
+_For clarity, some Zoe API details are ommitted from this figure._
+
+The committee participant then instructs their smart wallet to
+redeem the charter invitation to get a capability to put a question using `VoteOnParamChange`,
+using `v0-accept-charter` to identify this offer:
+
+```js
+test.serial('Voter0 accepts charter, committee invitations', async t => {
+...
+  await victor.acceptCharterInvitation('v0-accept-charter');
+...
+});
+```
+
+<img src="./assets/gov-param-4-accept.svg"
+  style="border: 2px solid" width="600" />
+
+To exercises their `VoteOnParamChange` capability,
+the committee participant makes a _continuing invitation_,
```

Comment:
Is there an explanation of continuing invitations somewhere to link to? Is it in the queue?

Reply:
> Is there an explanation of continuing invitations somewhere to link to?

Not really; the markup here is intended to signal introducing / defining the term.

There should be.

> Is it in the queue?

#667 about continuing invitations was opened Apr 2022, but it's not scheduled.

#878 about invitation specs is scheduled... sort of. One kind of invitation spec is a continuing invitation, so it's possible #667 will be addressed along with it.

Reply:
I can add a glossary entry pointing here for now

Reply:
My dislike for novel info in the glossary prompted me to go ahead and write up continuing invitations and the rest of invitation specs, based on...

 - https://github.com/Agoric/agoric-sdk/discussions/8082

So now this PR closes #878, #667, #952, #749 :)